tasks/main.yml

```diff
@@ -48,15 +48,15 @@
   when:
     - assessment_tool_rust or assessment_tool_cargo_packages is defined
 
-- name: Install Python 2 if necessary
-  ansible.builtin.include_tasks: install_python2.yml
-  when:
-    - assessment_tool_python2
-
-- name: Create a Python virtualenv if necessary
-  ansible.builtin.include_tasks: create_python_venv.yml
+- name: Install Python and create Python virtualenv
   when:
     - assessment_tool_pip_packages is defined or assessment_tool_pip_requirements_file is defined
```

Comment:
This logic now gates whether or not Python (both 2 and 3) is installed at all. Does this match the behavior we expect?

Reply:
The intent is that if the user does not specify a list of `pip` packages to install or a requirements file, then we don't create a virtual environment and hence have no reason to install Python.

Would you prefer that we (choose all that apply, and feel free to add more options if necessary):
- [ ] Also install Python if `assessment_tool_python_install_development_dependencies` or `assessment_tool_python2` is defined and set to `true`?
- [ ] Also install Python (and perhaps create an empty venv) if a _new_ role variable, say, `assessment_tool_is_python` is defined and set to `true`?

Reply:
I went ahead and created cisagov/kali-packer#151 to identify any edge cases that we may have missed.

Reply:
I suppose the latter makes the most sense to me? My thinking is that a project with a Python script that has no external package dependencies would no longer be guaranteed to work with this change in place. I'm not sure how much it technically matters but I see it as breaking behavior.

Reply:
Please see commit b7305d4.